apps/aecore/src/aec_peers.erl

```diff
@@ -505,6 +505,8 @@ ping_peer(Peer) ->
                 {error, protocol_violation} ->
                     block_peer(Uri);
                 _ ->
+                    %% If we ping a peer with wrong API version, time out in aue_request 
```

Comment:
I do not expect aeu_requests:ping to time out rather to either return an error value or err.